course_content/iris_course/4.Joining_Cubes_Together.ipynb

```diff
@@ -50,7 +50,7 @@
    "source": [
     "----\n",
     "\n",
-    "## 3.1 Merge<a id='merge'></a>\n",
```

Comment:
I'm really confused by this! I swear we already fixed this!
I've noticed a few things that I thought we had fixed still be there...